lib/mongodb.js

```diff
@@ -797,6 +797,33 @@ function hasNearFilter(where) {
   return false;
 }
 
+MongoDB.prototype.buildLookup = function (relation, include, model) {
+  var self = this;
```

Comment:
@shalkam If I understand correctly the options are applied on per query not per model right?
A model could have both `hasMany` and `belongsTo` relation:
```
User hasMany Account
Uer belongsTo Group
```
`user.accounts` uses `hasMany`
while `user.group` uses `belongsTo`